bin/launcher.ps1

```diff
@@ -100,12 +100,22 @@ $tasks.Add('start', {
       AddRunnerInClassPath
       AddClassPathRequiredForExecution
     }
-    $env:CLASSPATH = $global:classpath
     if ("$env:GAUGE_DEBUG_OPTS" -ne "" ) {
       $debugArgs = "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=$env:GAUGE_DEBUG_OPTS,timeout=25000"
       Write-Output "`nRunner Ready for Debugging"
     }
-    & $javaCommand `"-Dfile.encoding=UTF-8`" $debugArgs $env:gauge_jvm_args com.thoughtworks.gauge.GaugeRuntime --start
+
+    $verison = (Get-Command java | Select-Object -ExpandProperty Version).toString() | Out-String
+
+    if ( "$verison" -lt "9" ) {
```

Comment:
This check should not be used to display that version 8 is not supported anymore.
Raise it all the way up to Gauge to inform the user to Upgrade Java or stick to older versions of the plugin

Reply:
@zabil 

```
➜  java-foo javac -version
javac 1.8.0_202
➜  java-foo gauge run specs
This version of the plugin does not support java version < 1.9
Please upgrade your java version or stick to an older version of gauge-java.
Error ----------------------------------

[Gauge]
Failed to start gauge API: Error occurred while waiting for runner process to finish.
Error : exit status 1

Get Support ----------------------------
	Docs:          https://docs.gauge.org
	Bugs:          https://github.com/getgauge/gauge/issues
	Chat:          https://spectrum.chat/gauge

Your Environment Information -----------
	darwin, 1.0.8, 3abcc41d
```

Reply:
Can you modify this to 

```
➜  java-foo gauge run specs

This version of gauge-java plugin does not support Java versions < 1.9
Please upgrade your Java version or use a version of gauge-java <= v0.7.4

Error ----------------------------------

[Gauge]
Failed to start gauge API: Error occurred while waiting for runner process to finish.
Error : exit status 1

Get Support ----------------------------
	Docs:          https://docs.gauge.org
	Bugs:          https://github.com/getgauge/gauge/issues
	Chat:          https://spectrum.chat/gauge

Your Environment Information -----------
	darwin, 1.0.8, 3abcc41d
```